plugins/ci/skills/slack-notify/slack-notify.sh

```
@@ -0,0 +1,111 @@
+      REVIEWER_MENTIONS="<@${SLACK_FALLBACK_USER_ID}>"
+    else
+      REVIEWER_MENTIONS="(none assigned)"
+    fi
+  fi
+  REVIEWER_MENTIONS=$(echo "$REVIEWER_MENTIONS" | sed 's/^ //')
+
+  local SLACK_PAYLOAD
+  SLACK_PAYLOAD=$(jq -n --arg title "$PR_TITLE" --arg url "$PR_URL" --arg reviewers "$REVIEWER_MENTIONS" --arg emoji "$SLACK_EMOJI" \
+    '{text: "\($emoji) *Jira Agent PR ready for review*\n:review: <\($url)|\($title)>\n:eyes: Reviewers: \($reviewers)"}')
+
+  [[ $- == *x* ]] && local _was_tracing=true || local _was_tracing=false
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)